nodejs/eks/examples/tags/package.json

```diff
@@ -6,9 +6,7 @@
     },
     "dependencies": {
         "@pulumi/pulumi": "^1.0.0-beta",
-        "@pulumi/aws": "^1.0.0-beta"
-    },
-    "peerDependencies": {
+        "@pulumi/aws": "^1.0.0-beta",
```

Comment:
While you are touching all these - may as well bump these all to `^1.0.0`?

Reply:
Good catch, thanks.